src/IFrameRoutes.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import {Route} from 'react-router-dom';
+import {ButtonHero} from './ButtonCatalog';
+import {CardHero} from './CardCatalog';
+import {CheckboxHero} from './CheckboxCatalog';
+import {ChipsHero} from './ChipsCatalog';
+import {DialogHero} from './DialogCatalog';
+import {DrawerHero} from './DrawerCatalog';
+import {ElevationHero} from './ElevationCatalog';
+import {FabHero} from './FabCatalog';
+import {IconButtonHero} from './IconButtonCatalog';
+import {ImageListHero} from './ImageListCatalog';
+import {LayoutGridHero} from './LayoutGridCatalog';
+import {ListHero} from './ListCatalog';
+import {MenuHero} from './MenuCatalog';
+import {RadioButtonHero} from './RadioButtonCatalog';
+import {RippleHero} from './RippleCatalog';
+import {SelectHero} from './SelectCatalog';
+import {SliderHero} from './SliderCatalog';
+import {SnackbarHero} from './SnackbarCatalog';
+import {SwitchHero} from './SwitchCatalog';
+import {TabsHero} from './TabsCatalog';
+import {TextFieldHero} from './TextFieldCatalog';
+import {TopAppBarHero} from './TopAppBarCatalog';
+import {TypographyHero} from './TypographyCatalog';
+import {LinearProgressHero} from './LinearProgressIndicatorCatalog';
```

Comment:
Nit: Put this in order? (I was wondering why it was missing until I got to the bottom.)

I assume theme is intentionally left out... are we expecting typography to be meaningful in this context?

Reply:
I removed Typography since it can also be rendered on the material io page easily.